src/FSharp.Data.DesignTime/FSharp.Data.DesignTime.fsproj

```diff
@@ -65,5 +66,8 @@
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.0.0.1" Condition="'$(TargetFramework)' == 'net45'" />
     <PackageReference Include="FSharp.Core" Version="4.2.3" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
+    <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.0" PrivateAssets="All" />
+    <PackageReference Include="SourceLink.Test" Version="2.8.0" PrivateAssets="all" />
```

Comment:
You don't need `SourceLink.Test` since you are calling `dotnet sourcelink test` yourself in the FAKE script.